docs/tests/sitemap.test.ts

```diff
@@ -0,0 +1,13 @@
+import * as fs from 'fs';
+
+test('Sitemap Snapshot', () => {
+  const sitemapData = fs.readFileSync('./public/sitemap.xml', 'utf8');
+  const regex = new RegExp('(?<=>).+?(?=</loc)', 'gmi');
```

Comment:
Can you add a comment explaining this regex?

Reply:
Good point. Updated.